docker/ci-py-go-themis.dockerfile

```diff
@@ -1,3 +1,86 @@
 FROM debian:buster
 
 SHELL ["/bin/bash", "-c"]
+
+# Install dependencies
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    apt-transport-https \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    gnupg \
+    libpq-dev \
+    libssl-dev \
+    openssl \
+    psmisc \
+    python python-setuptools \
+    python3 python3-setuptools python3-pip \
+    rsync \
+    sudo \
+    wget
+
+WORKDIR /root
+
+# Install libthemis
+RUN set -o pipefail && \
+    curl -sSL https://pkgs.cossacklabs.com/scripts/libthemis_install.sh | \
+        bash -s -- --yes
+
+# Include helpful scripts
+RUN mkdir /image.scripts
+COPY docker/_scripts/acra-build/install_go.sh /image.scripts/
+RUN chmod +x /image.scripts/*.sh
+
+# Install Go
+RUN GO_PREFIX_DIR=/usr/lib/go/1.14.9 \
+    GO_VERSION=1.14.9 \
+    GO_TARBALL_DIGEST=f0d26ff572c72c9823ae752d3c81819a81a60c753201f51f89637482531c110a \
+    GO_TARBALL_CLEAN=1 \
+    /image.scripts/install_go.sh
+
+RUN GO_PREFIX_DIR=/usr/lib/go/1.15.2 \
+    GO_VERSION=1.15.2 \
+    GO_TARBALL_DIGEST=b49fda1ca29a1946d6bb2a5a6982cf07ccd2aba849289508ee0f9918f6bb4552 \
+    GO_TARBALL_CLEAN=1 \
+    /image.scripts/install_go.sh
+
+ENV GOROOT="/usr/lib/go/1.15.2/go" GOPATH="/home/user/gopath" GO111MODULE="auto"
+# if you need Go, put `export PATH=$GOROOT/bin:$PATH` in you scripts
+# if you need a different Go version, set GOROOT to another /usr/lib/go/*/go before setting PATH
```

Comment:
> Why not add default Go version to PATH as well?

Because I though it's easier to do `PATH=$GOROOT/bin:$PATH`
than `PATH=$(echo $PATH | sed "s|/usr/lib/go/\d+\.\d+\.\d+/go/bin|$GOROOT/bin|")`
in case you need a custom Go version and don't wanna prepend many different `GOROOT/bin`s in `PATH`

Reply:
> Because I though it's easier to do PATH=$GOROOT/bin:$PATH

I mean, it's easier to *not have to do that at all* for plain `go` to just work, launching the default version of Go for the image. When a CI script wants to iterate through multiple versions, it can call them by absolute paths, or prepend `$GOROOT/bin` to PATH as necessary.